src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -12,25 +12,92 @@ import {
   countDecimalPlaces,
   extractColorValueForColors,
   findNiceIncrement,
+  isUnwrapEth,
+  isWrapEth,
   niceIncrementFormatter,
+  priceForAsset,
   trimTrailingZeros,
   valueBasedDecimalFormatter,
 } from '@/__swaps__/utils/swaps';
-import { ChainId } from '@/__swaps__/types/chains';
+import { ChainId, ChainName } from '@/__swaps__/types/chains';
 import { ParsedSearchAsset } from '@/__swaps__/types/assets';
 import { useColorMode } from '@/design-system';
 import { isSameAssetWorklet } from '@/__swaps__/utils/assets';
+import {
+  CrosschainQuote,
+  ETH_ADDRESS,
+  Quote,
+  QuoteError,
+  QuoteParams,
+  Source,
+  SwapType,
+  getCrosschainQuote,
+  getQuote,
+} from '@rainbow-me/swaps';
+import { chainNameFromChainIdWorklet } from '@/__swaps__/utils/chains';
+import { RainbowConfig, useRemoteConfig } from '@/model/remoteConfig';
+import { useAccountSettings } from '@/hooks';
+import { convertAmountToRawAmount, convertRawAmountToBalance, convertRawAmountToNativeDisplay } from '@/__swaps__/utils/numbers';
+import ethereumUtils from '@/utils/ethereumUtils';
+import { FormattedExternalAsset, fetchExternalToken } from '@/resources/assets/externalAssetsQuery';
+
+const QUOTE_REFETCH_INTERVAL = 5_000;
+const PRICE_REFETCH_INTERVAL = 10_000;
+
+export const DEFAULT_SLIPPAGE_BIPS = {
+  [ChainId.mainnet]: 100,
+  [ChainId.polygon]: 200,
+  [ChainId.bsc]: 200,
+  [ChainId.optimism]: 200,
+  [ChainId.base]: 200,
+  [ChainId.zora]: 200,
+  [ChainId.arbitrum]: 200,
+  [ChainId.avalanche]: 200,
+  [ChainId.blast]: 200,
+};
+
+export const DEFAULT_SLIPPAGE = {
+  [ChainId.mainnet]: '1',
+  [ChainId.polygon]: '2',
+  [ChainId.bsc]: '2',
+  [ChainId.optimism]: '2',
+  [ChainId.base]: '2',
+  [ChainId.zora]: '2',
+  [ChainId.arbitrum]: '2',
+  [ChainId.avalanche]: '2',
+  [ChainId.blast]: '2',
+};
+
+const slippageInBipsToString = (slippageInBips: number) => {
+  'worklet';
+  return (slippageInBips / 100).toString();
+};
+
+export const getDefaultSlippage = (chainId: ChainId, config: RainbowConfig) => {
+  'worklet';
+
+  const chainName = chainNameFromChainIdWorklet(chainId) as
+    | ChainName.mainnet
```

Comment:
prob worth pulling this out into a supportedChains type or similar

Reply:
added a new ticket: https://linear.app/rainbow/issue/APP-1348/cleanup-supportedchains-type